synth.py

```diff
@@ -71,7 +71,6 @@
 # ----------------------------------------------------------------------------
 templated_files = common.py_library(
     samples=False,  # set to True only if there are samples
-    unit_test_python_versions=["3.6", "3.7", "3.8"],
     system_test_python_versions=["3.7"],
```

Comment:
If we are templating this, your change is going to get overwritten. Should this line be updated not removed?

Reply:
The templated change will Do the Right Thing(TM).

Reply:
@crwilcox PTAL (if the value is not set, the defaults are correct).